packages/components/src/components/multi-select/multi-select/multi-select.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { MultiSelect } from './multi-select';
+import * as multiSelectUtils from './multi-select-utils';
+import * as isWithinFormUtils from '../../../utils/form/isWithinForm';
+
+const initComponent = (): MultiSelect => {
+  const component = new MultiSelect();
+  component.host = document.createElement('p-multi-select');
+  component.host.attachShadow({ mode: 'open' });
+  return component;
+};
+
+describe('connectedCallback', () => {
+  it('should add event listener and set is within form', () => {
+    const component = initComponent();
+    const isWithinFormSpy = jest.spyOn(isWithinFormUtils, 'isWithinForm');
+    const eventListenerSpy = jest.spyOn(document, 'addEventListener');
+    // const spy = jest.spyOn(multiSelectUtils, 'initNativeSelect');
+    component.connectedCallback();
+    expect(isWithinFormSpy).toBeCalledTimes(1);
+    expect(eventListenerSpy).toBeCalledTimes(1);
```

Comment:
Please check for correct name and handler.
Also the result of isWithinForm isn't checked, therefore the test description doesn't match what is happening here.